robot/camera/src/main.cpp

```diff
@@ -0,0 +1,25 @@
+﻿#include "VideoStream.hpp"
+
+int main(int argc, char *argv[])
+{
+    VideoStream stream("Test stream");
+
+    try 
+    {
+        // This is an example pipeline.
+        // Modify it to your liking
+        stream.SetPipeline("v4l2src device=/dev/video0 ! videoconvert ! autovideosink");
+        stream.ListenOn("127.0.0.1", 8080);
+
+        while (stream.IsListening()) 
+        {
+            std::this_thread::sleep_for(std::chrono::milliseconds(500));
```

Comment:
dodał bym te rzeczy do environment variables

Reply:
Ok, wstępnie to były takie "suche" wartości na sztywno, aby dać przykład działania apki. Myślę, że dodam potem jakiś sposób, aby dynamicznie to ustalać... Na teraz dodałbym zmienne, aby nie było magicznych zmiennych